thaw/src/select/mod.rs

```diff
@@ -1,177 +1,89 @@
+pub(crate) mod raw;
 mod theme;
 
 pub use theme::SelectTheme;
 
-use crate::{theme::use_theme, Theme};
 use leptos::*;
-use std::hash::Hash;
-use thaw_components::{Binder, CSSTransition, Follower, FollowerPlacement, FollowerWidth};
-use thaw_utils::{class_list, mount_style, Model, OptionalProp};
+use std::{hash::Hash, rc::Rc};
+use thaw_utils::{Model, OptionalProp};
 
-#[derive(Clone, PartialEq, Eq, Hash)]
+use crate::{select::raw::RawSelect, TagVariant};
+
+#[slot]
+pub struct SelectLabel {
+    children: ChildrenFn,
+}
+
+#[derive(Clone, Default, PartialEq, Eq, Hash)]
 pub struct SelectOption<T> {
     pub label: String,
     pub value: T,
+    pub variant: TagVariant,
```

Comment:
There will be destructive updates here, so maybe we can extract it as MultiSelectOption better.